src/components/ContentSidebar/SidebarVersions.js

```diff
@@ -6,18 +6,19 @@
 
 import React from 'react';
 import VersionHistoryLink from 'box-react-ui/lib/features/item-details/VersionHistoryLink';
-import SidebarSection from './SidebarSection';
+import withErrorHandling from './withErrorHandling';
 import type { FileVersions } from '../../flowTypes';
 
 type Props = {
     onVersionHistoryClick?: Function,
-    versions: FileVersions
+    versions?: FileVersions
 };
 
-const SidebarVersions = ({
+export const SidebarVersions = ({
```

Comment:
All exports should be at the bottom, so in this case

export { SidebarVersions as SidebarVersionsComponent };
export default withErrorHandling(SidebarVersions);

Reply:
Or something like


const SidebarVersionsComponent = ...
const SidebarVersions = withErrorHandling(SidebarVersions);

export { SidebarVersionsComponent };
export default SidebarVersions;

Reply:
There is also this style

export { SidebarVersions as default,  SidebarVersionsComponent };